engine/src/main/scala/cromwell/engine/workflow/WorkflowDockerLookupActor.scala

```diff
@@ -249,7 +251,7 @@ object WorkflowDockerLookupActor {
   final case class WorkflowDockerTerminalFailure(reason: Throwable, request: DockerHashRequest) extends WorkflowDockerLookupResponse
 
   def props(workflowId: WorkflowId, dockerHashingActor: ActorRef, startMode: StartMode, databaseInterface: SqlDatabase = SingletonServicesStore.databaseInterface) = {
-    Props(new WorkflowDockerLookupActor(workflowId, dockerHashingActor, startMode, databaseInterface))
+    Props(new WorkflowDockerLookupActor(workflowId, dockerHashingActor, startMode, databaseInterface)).withDispatcher(EngineDispatcher)
```

Comment:
- Yes
- Perhaps? Slick itself has its own threadpool

This is all a little academic right now as no one bothers to tune these dispatchers for their installation so in some ways they're not really all that useful. CaaS will give us an installation that'll be worth tuning though. I'd rather not prematurely add a ton of granularity that we might never use, let's dogfood what's there over the next few months and use that to drive how to change